### PR TITLE
arm64 build targets for released cli

### PR DESCRIPTION
**Pull request by faddat**
* add linux-arm64
* add darwin-arm64

closes: #918

### CODE REVIEW

**Comment by faddat** [comment]
Hi, I think you may need to enable CI for this.

**Comment by faddat** [comment]
https://dev.azure.com/enigmampc/Secret%20Network/_build/results?buildId=2566&view=logs&j=12f1170f-54f2-53f3-20dd-22fc7dff55f9&t=5caf77c8-9b10-50ef-b5c7-ca89c63e1c86&l=83

**Comment by assafmo** [comment]
Hi @faddat, thanks for this!
I think that the CI failing is not related to this PR.

**Comment by Cashmaney** [comment]
Yeah, the CI will not run properly on targets coming from forks, but it should be all good. We can merge this and add the target to the runner matrix

File: .github/workflows/release.yaml [hunk 1]
```diff
@@ -135,6 +135,7 @@ jobs:
         run: |
           make build_cli
           cp "secretcli" "secretcli-$RUNNER_OS"
+          make build_linux_arm64_cli
```
Comment (Cashmaney) [inline .github/workflows/release.yaml:138]: Copying from the other comment:

The arm64 build was added to a line that runs on each build target in the matrix. A better way is to add the target to the matrix instead, that way it gets built and uploaded/released automatically. IMO linux-arm64 is supported, and there should be a mac m1 (darwin-arm64) target supported on GH actions, though I'm not 100% about that

**Comment by Cashmaney** [comment]
Closing this in favor of #925

**Pull request closed without merging**